### PR TITLE
added conifg and docker-compose

### PR DESCRIPTION
**Pull request by akp111**

### CODE REVIEW

File: config/P/config.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "snowman-api-enabled": false,
+    "coreth-admin-api-enabled": false,
+    "coreth-admin-api-dir": "",
+    "state-sync-enabled": false,
```
Comment (0xabcdefgh) [inline config/P/config.json]: Why not we are enabling this, it is enabled in c-chain ?

Comment (0xabcdefgh) [reply config/P/config.json]: P-chain is the one who is going to create blocks and validate so we should do state sync don't we ? Maybe read more from here to make a informative decision https://docs.avax.network/nodes/configure/chain-config-flags#state-sync-enabled-boolean

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+services:
+  go-flare:
+    image: flarefoundation/go-flare
+    container_name: go-flare-node
+    # environment:
```
Comment (0xabcdefgh) [inline docker-compose.yml]: Don't need to give the environment ? is it automatically takes the local config and start running ?

File: config/P/config.json (hunk 1 above)
Comment (akp111) [reply config/P/config.json]: The thing is, it starts syncing with the testnet. And unless until it is completely synced, it wont allow me to query through the node

File: docker-compose.yml (hunk 2 above)
Comment (akp111) [reply docker-compose.yml]: Yeah, without env variables, it takes up default values.

File: config/P/config.json (hunk 1 above)
Comment (0xabcdefgh) [reply config/P/config.json]: Okay then keep as it is.

File: docker-compose.yml (hunk 2 above)
Comment (0xabcdefgh) [reply docker-compose.yml]: And does the node automatically cleaned up from the local system when the test process get shut down ?

**Review by avinash-onebyte — APPROVED** [review]

**Pull request merged**